packages/protocol/contracts/stability/StableTokenRegistry.sol

```diff
@@ -0,0 +1,115 @@
+pragma solidity ^0.5.13;
+
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+import "../common/Initializable.sol";
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+import "../common/interfaces/IRegistry.sol";
+
+/**
+ * @title contract that lists what stable coins are deployed as part of Celo's Stability protocol.
+ */
+contract StableTokenRegistry is Initializable, Ownable {
+  using SafeMath for uint256;
+  mapping(bytes => bytes) public stableTokens;
+  bytes[] public fiatTickers;
+
+  /**
+   * @notice Sets initialized == true on implementation contracts
+   * @param test Set to true to skip implementation initialization
+   */
+  constructor(bool test) public Initializable(test) {}
+
+  /**
+   * @notice Used in place of the constructor to allow the contract to be upgradable via proxy.
+   * @param fiatTicker Collection of fiat currencies issued already.
+   * @param stableTokenContractName Collection of stable token smart contract names.
+   */
+  function initialize(bytes calldata fiatTicker, bytes calldata stableTokenContractName)
+    external
+    initializer
+  {
+    _transferOwnership(msg.sender);
+    addNewStableToken(bytes("USD"), bytes("StableToken"));
+    addNewStableToken(bytes("EUR"), bytes("StableTokenEUR"));
+    addNewStableToken(bytes("BRL"), bytes("StableTokenBRL"));
+    addNewStableToken(fiatTicker, stableTokenContractName);
```

Comment:
I think this optional? If we don't have a stable to deploy what do we put in here?

Reply:
yes, for now it's not optional

Reply:
Better to make this optional